### ISSUE TITLE
`.as_matrix()` removed from pandas now is `.values()`

### ISSUE THREAD

**Issue by Carreau** [OlivierGarciaDev/fanalysis#1]
Hi, 

It looks like `as_matrix()` has been deprecated since pandas 0.23 and finally been removed. 

Would you be open to a pull request that update the code, and if so will you publish an updated version on PyPI ? 

Thanks.

**Comment by fcarpentey75012** [comment]
Hi Carreau, 

You can replace as_matrix() by .values() and keep the all code

**Comment by Carreau** [comment]
> You can replace as_matrix() by .values() and keep the all code

Yes, that's what my title say, but the code here does use `.as_matrix()` in the test and example. I'm asking if it's ok to send a PR to update.

**Comment by fcarpentey75012** [comment]
Yes, i'm sorry. I didn't read the title..
